doc/source/Platforms.rst

```diff
@@ -342,6 +342,7 @@ This instructions are meant to be a reference that users can follow to set up th
    brew install python
    brew install git
    brew install git-lfs
+   brew install git-flow-avh
```

Comment:
What is that?

Reply:
Same question!

Reply:
Do we need this package? I don't know what it is doing and I didn't need it to build any of our codes.

If we need it, it needs to be added on all platforms and in the generic Linux instructions, too. If it is not mandatory, then maybe add a separate `code_block`, making it optional and providing a reason why this is good to have (and do the same for Linux)?

Reply:
This is the git flow package that we have been using for several years. This add macros to git that help manage the "git flow" methodology: https://nvie.com/posts/a-successful-git-branching-model/

Reply:
git-flow-avh is handy for managing feature branches and PRs for developers, and as you say is not necessary for building the code. We recommend it in the JEDI ReadTheDocs documentation and use the "git flow" commands in many tutorial examples including advocating it in our JEDI Academies.

I can see the point that adding the git-flow-avh package to the list of homebrew installs is introducing a JEDI specific need in the broader scope documentation. Plus you don't need it for doing the build.

However, the use of "git flow" perhaps goes beyond JEDI. The very next thing many people will want to do after running the build (if git-flow-avh is not mentioned here) will be to install git-flow-avh and start using it.

I'm leaning toward removing git-flow-avh from this list, and leaving it up to the different groups using spack-stack to advocate (or not advocate) for it.

What do you think?

Reply:
I concur. We can always keep the recommendation in the JEDI docs. But I am also interested in learning more about it.